internal/tools/grantvmgroupaccess/main.go

```diff
@@ -3,18 +3,51 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"os"
 
-	"github.com/Microsoft/go-winio/pkg/security"
+	"github.com/Microsoft/hcsshim/internal/security"
 )
 
 func main() {
-	if len(os.Args) != 2 {
-		fmt.Fprintln(os.Stderr, "Usage: grantvmgroupaccess.exe file")
+	readFlag := flag.Bool("read", false, "Grant GENERIC_READ permission (DEFAULT)")
+	writeFlag := flag.Bool("write", false, "Grant GENERIC_WRITE permission")
+	executeFlag := flag.Bool("execute", false, "Grant GENERIC_EXECUTE permissions")
+	allFlag := flag.Bool("all", false, "Grant GENERIC_ALL permissions")
+
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "\nUsage of %s:\n\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "    %s [flags] [file]\n\n", os.Args[0])
```

Comment:
I think technically it should be `%s [flags] file` as `file` is not optional. But that is a super NIT :)